recipes/jwt-cpp/all/conanfile.py

```diff
@@ -16,19 +16,24 @@ class JwtCppConan(ConanFile):
     def _source_subfolder(self):
         return "source_subfolder"
 
+    @property
+    def _supports_generic_json(self):
+        return tools.Version(self.version) >= "0.5.0"
```

Comment:
Should we add options for `JWT_DISABLE_PICOJSON` or `JWT_DISABLE_BASE64`?

Reply:
The problem is it does not affect the output package, consumers can easily add those defined on their own

Reply:
Ok, it makes sense.